### ISSUE TITLE
Can't download the video "..........", probably is only for premium users

### ISSUE THREAD

**Issue by shasureshkumar** [mclmza/linkedin-learning-downloader#17]
Hi,
I am having 2 linkedin accounts viz. personal and official one for linkedin learning.
When I try to give my personal account credentials I couldn't able to download the videos and getting the below error.
`Can't download the video "..........", probably is only for premium users`

But I can view the same video if I login with the same credentials, can you please help me on the same?
Is it because of two different accounts or premium users issue?

Thanks in advance.

**Comment by duracotton** [comment]
I get the same message, although I am premium user.

**Comment by candyalladi** [comment]
even I got the same issue. Seems the code works only for demo videos or only for introduction videos but not for Premium videos. Let me know working code to download any videos from linked in.

**Comment by candyalladi** [comment]
Tried many times. Still same issue. Unable to download videos with same error

**Comment by NawTan** [comment]
Same issue here for all videos and usage of premium account.

**Comment by MRMuggles69** [comment]
Any solutions for this problem???

**Comment by ismailas001** [comment]
Dears,
I'm currently using Python 2.7.15 and i've the same issue>>  Can't download the video "..........", probably is only for premium users. appreciate your help .. 
Please note that! I've zero knowledge in Python.

**Comment by basilbaby81** [comment]
Do you have an active subscription ?
If Yes .. share your config file after removing your username and password

**Comment by ismailas001** [comment]
Yes, i have active subscription / user premium account and i can play all the videos course directly from Linkedin Learning without any issue but when i used python script it seems it's downloading only the demo videos.

here is the config file as you requested:
#!/usr/bin/python

USERNAME = ''
PASSWORD = ''
BASE_DOWNLOAD_PATH = 'C:\TC' #use "/" as separators
SUBS = True
COURSES = [
    'integrating-tableau-and-r-for-data-science'
]

**Comment by ismailas001** [comment]
any update! i don't think my issue is with the config file.. appreciate your help

**Comment by ArthurEzenwanne** [comment]
The issue is **not** with the config file. Most probably its a Linkedin problem or network (IMO). I notice that rarely; maybe one video in like 30 ~ 50 videos.

**Comment by mauriziovitale** [comment]
@shasureshkumar do you have an organization account?
